torch/testing/_internal/opinfo/core.py

```diff
@@ -2061,7 +2064,6 @@ def __init__(
         error_inputs_func=None,
         lhs_make_tensor_kwargs=None,
         rhs_make_tensor_kwargs=None,
-        promotes_int_to_float=False,  # Set to true if the op promotes integer inputs to float
```

Comment:
This is promoted (pun intended) to `OpInfo` from `BinaryUfuncInfo`, to avoid duplication.